web/apps/labelstudio/src/pages/CreateProject/Config/Preview.jsx

```diff
@@ -10,9 +9,11 @@ import { useAPI } from "../../../providers/ApiProvider";
 
 const configClass = cn("configure");
 
+const loadDependencies = async () => import("@humansignal/editor");
+
 export const Preview = ({ config, data, error, loading, project }) => {
-  const LabelStudio = useLibrary("lsf");
   const lsf = useRef(null);
+  const [resolvingEditor] = useState(loadDependencies);
```

Comment:
it's not used as a proper state, better to use `useMemo()` maybe and it will be called only once?